bpf/bpf_lxc.c

```diff
@@ -812,7 +812,9 @@ int tail_handle_arp(struct __ctx_buff *ctx)
 #endif /* ENABLE_ARP_RESPONDER */
 #endif /* ENABLE_IPV4 */
 
-/* Attachment/entry point is ingress for veth, egress for ipvlan. */
+/* Attachment/entry point is ingress for veth, egress for ipvlan.
+ * It corresponds to packets entering the container.
```

Comment:
Shouldn't `It corresponds to packets entering the container.` rather say `It corresponds to packets coming from the container.`?